lib/tools/adb-emu-commands.js

```diff
@@ -5,42 +5,37 @@ import path from 'path';
 
 let emuMethods = {};
 
-emuMethods.isEmulatorConnected = async function (udid) {
+emuMethods.isEmulatorConnected = async function () {
   let emulators = await this.getConnectedEmulators();
-  if (!emulators.length) {
-    return false;
-  }
-  for (let emulator of emulators) {
-    if (emulator.udid === udid) {
-      return true;
-    }
+  return !!_.find(emulators, (x) => x && x.udid === this.curDeviceId);
+};
+
+emuMethods.checkEmulatorConnected = async function () {
```

Comment:
since this throws, maybe we should call it `verifyEmulatorConnected` rather than `check`, since I'd expect a verification to throw but not a check?